airflow-core/src/airflow/ui/public/i18n/locales/de/dashboard.json

```
@@ -1,4 +1,10 @@
 {
+  "favorite": {
+    "favoriteDags_one": "Erstes favorisiertes DAG",
```

we say "das Dag"? I think I used to say "der Dag", as in "der Graph" ... but i might be off here.

---

Good point... never thought of it... but feels like you are right!
```suggestion
    "favoriteDags_one": "Erster favorisierter DAG",
```